summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormTopologyBuilder.scala

```diff
@@ -21,43 +21,62 @@ import scala.reflect.ClassTag
 case class StormTopologyBuilder(options: Map[String, Options], jobId: JobId, stormDag: Dag[Storm]) {
   @transient private val logger = LoggerFactory.getLogger(classOf[StormTopologyBuilder])
 
-  def build(): StormTopology = {
-    val topologyBuilder = new TopologyBuilder
-    stormDag.nodes.foreach {
-      case summerNode: SummerNode[Storm] => scheduleSummerBolt(summerNode, topologyBuilder)
-      case flatMapNode: FlatMapNode[Storm] => scheduleFlatMapper(flatMapNode, topologyBuilder)
-      case sourceNode: SourceNode[Storm] => scheduleSpout(sourceNode, topologyBuilder)
-    }
-    topologyBuilder.createTopology()
-  }
+  def apply: StormTopology = scheduleEdges(scheduleNodes).build(jobId)
```

Comment:
I feel build is a more intuitive name here, it's a very standard method name on builders to build what they are supposed to build.